src/evaluations/run_evaluation.py

```diff
@@ -58,69 +66,79 @@
     'num_runs', None, 'The number of runs per scenario.', lower_bound=1)
 
 # Analysis parameters.
-flags.DEFINE_float(
-    'error_margin', 0.05,
-    'a positive number setting the upper bound of the error. '
-    'By default, set to 0.05.',
-    lower_bound=0)
-flags.DEFINE_float(
-    'proportion_of_runs', 0.95,
-    'a number between 0 and 1 that specifies the proportion of runs. '
-    'By default, set to 0.95.',
-    lower_bound=0, upper_bound=1)
+flags.DEFINE_list(
+    'error_margin', '0.05',
+    'A comma-separated list of positive numbers setting the upper bound of '
+    'the error. By default, use 0.05.')
+flags.DEFINE_list(
+    'proportion_of_runs', '0.95',
+    'A comma-separated list of a number between 0 and 1 that specifies the '
+    'proportion of runs. By default, use 0.95.')
+flags.DEFINE_integer('boxplot_xlabel_rotate', 0,
+                     'The degrees of rotation of the x labels in the boxplot.')
 flags.DEFINE_integer('boxplot_size_width_inch', 12,
                      'The widths of the boxplot in inches.')
 flags.DEFINE_integer('boxplot_size_height_inch', 6,
                      'The widths of the boxplot in inches.')
 
+required_flags = ('evaluation_config', 'sketch_estimator_configs',
+                  'evaluation_run_name', 'num_runs')
```

Comment:
Shall we also add `evaluation_out_dir` to required?

Reply:
sure good idea!